guides/Front-end-strategy.md

```diff
@@ -0,0 +1,44 @@
+# Front-end strategy
+
+### What we use
+####Foundation 6 framework
```

Comment:
A space after `####`? Also a blank line before and after a header?

PS: We need a Markdown lint.